### ISSUE TITLE
Block model being tesselated causes a nullPointerException.

### ISSUE THREAD

**Issue by Octaeon** [mickelus/tetra#130]
What happened:
I was making a little modpack, started a test world, and it immediately crashed as I was looking around. Looking at the crash log, I can see that a 'forged platform slab' caused this.

Here's the crash log:
[https://hastebin.com/vuniwutiko.rb](https://hastebin.com/vuniwutiko.rb)

The mods are listed in the crash log above, but here are the details:
- Forge 1.14 28.2.1
- Tetra 1.14 1.21.0
- Tetra configuration: I didn't tweak anything - it was the first time starting the game with this mod setup, or even with the mod itself.

**Comment by mickelus** [comment]
Is that the minimal set of mods required to produce this issue?

**Comment by Octaeon** [comment]
Not really... I don't really know what caused this issue, and I simply noticed that tetra came up as the reason. It may be due to some other mods interfering with block tesselation, and I don't really know... If anything, these mods are probably the performance improvement ones(optifine, betterfps, etc.) I'm gonna look a bit into it.

**Comment by AudibleWallpaper** [comment]
Having run into the same nullPointerException ( https://hastebin.com/icusedosew.rb ), I've started doing some digging. I've created a fresh profile with only tetra loaded and the Exception does not occur even when forged platform slabs are for sure being generated. I have since gone through Octaeons and my mod lists and found the common mods and added only them to the fresh profile. I was able to deduce the problem to be MixinBootstrap (https://github.com/LXGaming/MixinBootstrap/releases/download/v1.0.0/MixinBootstrap-1.0.0.jar), a mod launcher service for BetterFoliage. The crash world not occur without the file loaded. Hope this helps!
